### DISCUSSION TITLE
Italian translation standards

### DISCUSSION THREAD

**Discussion by GitGianluc** [URUWorks/TeroSubtitler#269] [Show and tell]
@bovirus Thank you very much for joining the Italian “translation team” for Tero Subtitler's interface and for making numerous improvements to my initial version! However, I would like to define and agree on some conventions that have been used since the beginning of the translation and that I would like to see maintained with consistency.

**PUNCTUATION**
- For ellipses => … (single glyph, NOT three separate dots)
- For quotation marks => “…” or ‘…’ (English quotation marks, NOT apostrophes)
- For apostrophes => ’ (right single quotation mark, NOT apostrophe)

**LEXICON**
- _long_ = «esteso» (NOT «lungo»)
- _line_ = «riga» (NOT «linea»)

**STYLE**
- It's okay to save on articles, but without losing the sense of the sentence in Italian («Vai al punto finale elemento attuale» sounds quite odd…)

Let's use this thread, if necessary, to expand this glossary or to discuss any doubts. Se poi vogliamo proseguire la discussione in italiano, tanto meglio! 😄 

Thanks again and see you soon!

**Comment by bovirus** [comment]
@GitGialnluc

Some personal feedback

**PUNCTUATION**

For ellipses => … (single glyph, NOT three separate dots)
I believe that if the original is ".... " it's better to use it.
If for you appear better we can change with single glyph 

For quotation marks => “…” or ‘…’ (English quotation marks, NOT apostrophes)
For me 'xxx' appear as translation style more modern and lighter then "xxx".

For apostrophes => ’ (right single quotation mark, NOT apostrophe)
OK

**LEXICON**

long = «esteso» (NOT «lungo»)
OK

line = «riga» (NOT «linea»)
For me "line" is linea" and "row" is riga"

**STYLE**

It's okay to save on articles, but without losing the sense of the sentence in Italian («Vai al punto finale elemento attuale» sounds quite odd…)

For me it's important separate consideration between menu option strings and option explanation strings.
In the menu most of the time the article is not used and is not necessary
For option explanation strings (quite longer then menu option) the use of article could be considered.

Generally pelase consdier that the italian strings are longer then english strings and in the UI you can have some limitations about space available for the striung.
